### DISCUSSION TITLE
Unable to load libhdfs | mlflow 1.27.0

### DISCUSSION THREAD

**Discussion by kharepraveenLearning** [mlflow/mlflow#6837] [Q&A]
Hello All,

I am using mflow 1.27.0  & facing issue in storing deep learning ML model artefacts  into HDFS location. below is some details

[ERROR | myDriver.py:47] > Unable to load libhdfs: ./libhdfs.so: cannot open shared object file: No such file or directory

I am starting server using below command -
**mlflow server --backend-store-uri sqlite:///*****.db --default-artifact-root hdfs:///******:9000/artifact --host 0.0.0.0**

mlflow command  which is causing issue is - 
 **mlflow.keras.log_model(model,artifact_path="models",registered_model_name=model_name)**

The libhdfs file is available under  /opt/cloudera/parcels/***/lib64/

Can anyone please suggest how to fix this.

Thank You.
Praveen

**Comment by JHibbard** [comment]
Hi Praveen,

This sounds like a HDFS issue instead of an MLflow issue. If you're able to share your code and details about your environment/setup, I can help you start troubleshooting though.

Hope that helps!
Jim
